go.mod

```diff
@@ -3,29 +3,29 @@ module sigs.k8s.io/custom-metrics-apiserver
 go 1.17
```

Comment:
while we are at it, would you mind also requiring golang 1.18?

Reply:
we can give a try, but in KEDA we still use 1.17 xD
Should I update golang in this PR?

Reply:
on it

Reply:
done! PTAL